contrib/scripts/conventional-commits

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
```

Comment:
was this code taken from somewhere? I saw you modified it to fit DragonflyDb but I wonder if it makes sense to reference here the origin .

Reply:
This code was pulled this from one of my repo's I have not opened sourced yet...

"Inspired from" sources would be:
https://github.com/compilerla/conventional-pre-commit
https://gist.github.com/qoomon/5dfcdf8eec66a051ecd85625518cfd13

Not sure if this merits linking ... seems distinct enough to me; can add references if you like...